extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureDataSegmentPusher.java

```diff
@@ -153,14 +153,7 @@ public DataSegment push(final File indexFilesDir, final DataSegment segment, fin
   @Override
   public Map<String, Object> makeLoadSpec(URI uri)
   {
-    return ImmutableMap.of(
-        "type",
-        AzureStorageDruidModule.SCHEME,
-        "containerName",
-        segmentConfig.getContainer(),
-        "blobPath",
-        uri.toString()
-    );
+    return makeLoadSpec(uri.toString());
```

Comment:
It looks like hadoop indexing would still call this method so it might have issues with funny characters, but I don't think it is worth refactoring to fix the issue at this point.